DesignSafe-CI/portal
# nav

## Overview: ##
Add/Remove options to the dropdown list of "Data Types" for Other projects.
 
## PR Status: ##


## Related Jira tickets: ##

* [DES-2340](https://jira.tacc.utexas.edu/browse/DES-2340)

## Summary of Changes: ##
The two files changed: 
designsafe/apps/api/publications/search_utils.py
designsafe/static/scripts/projects/components/manage-project/project-form-options.json

## Testing Steps: ##
Three places to see changes: 
1. My Projects —> Other Project —> Edit Project —> Data Type dropdown
2. My Projects —> Other Project —> Publication Preview —> Amend —> Edit Project Metadata —> Data Type dropdown
3. Publications Search —> More Options —> Other —> Data Type dropdown

## UI Photos:
Before: 
![Screen Shot 2022-10-17 at 3 24 51 PM](https://user-images.githubusercontent.com/35277477/198106471-327efd7e-3154-4f50-b28c-3b12af374e91.png)

After:
![Screen Shot 2022-10-26 at 1 23 44 PM](https://user-images.githubusercontent.com/35277477/198106541-e202d530-2ab9-4da5-9af6-7ad0c973d022.png)

## Notes: ##
search_utils.py is used in designsafe/apps/api/publications/operations.py, however I didn't see how changing search_utls.py affected the page. I made the changes to search_utils.py anyway.
